website/content/en/docs/kubebuilder/advanced-topics.md

```diff
@@ -115,13 +115,8 @@ func main() {
 
 ### Metrics
 
-> **// TODO:** Update the [metrics doc](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/golang/metrics/operator-sdk-monitoring.md) since it doesn't match the default scaffolded by kubebuilder anymore.
-
-To learn about how metrics work in the Operator SDK read the [metrics section][metrics_doc] of the user documentation.
-
-#### Default Metrics exported with 3rd party resource
+To learn about how metrics work in the Operator SDK read the [metrics section][metrics_doc] of the Kubebuilder documentation.
 
-> **// TODO:** Remove this section since we're no longer scaffolding main.go to use the SDK's `GenerateAndServeCRMetrics()` util in `pkg/kube-metrics`.
 
 ### Handle Cleanup on Deletion
 
```

Comment:
@theishshah There's still a **TODO** here in the finalizer section for updating the finalizer section of this doc with the new var names and signatures of the kubebuilder controller.
> **// TODO:** Update finalizer reconcile code for kubebuilder's default reconciler imports and variable names